src/amp-form-error.html

```diff
@@ -1,29 +1,23 @@
+<!---{
+  "hideCode": true,
+  "hidePreview": true
+}--->
 <!doctype html>
 <html ⚡>
 <head>
+  <link rel="canonical" href="https://ampbyexample.com/amp-form-error/">
   <meta charset="utf-8">
-  <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
-  <script async src="https://cdn.ampproject.org/v0.js"></script> 
-    <style amp-custom>
-        p {
-            margin-top: 4px;
-            font-size: 20px;
-            color: #dc4e41;
-        }
-        #title {
-            text-align: center;
-            font-family: Roboto,Helvetica,Arial,sans-serif;
-            font-size: 34px;
-            line-height: 40px;
-        }
-    </style>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>  
 </head>
 <body> 
-  <h3 id="title">
-        Error page for amp-form
-  </h3>
-  <p>Thanks for trying amp-form demo with an error response. This page has been opened because you chose the amp-form with HTTP request. You can go back to the previous page.</p>
+  <!--
+  #### Error page for amp-form
+
+  Thanks for trying amp-form demo with an error response. This page has been opened because you chose the amp-form with HTTP request. You can go back to the previous page.
```

Comment:
Use the same sentence as in the other page: "Error! Thanks for trying amp-form demo. This page has been opened because you chose the amp-form with HTTP request."